docassemble/ALToolbox/misc.py

```diff
@@ -84,24 +84,6 @@ def collapse_template(template, classname=None):
 <a class="collapsed" data-bs-toggle="collapse" href="#{}" role="button" aria-expanded="false" aria-controls="collapseExample"><span class="pdcaretopen"><i class="fas fa-caret-down"></i></span><span class="pdcaretclosed"><i class="fas fa-caret-right"></i></span> {}</a>
 <div class="collapse" id="{}"><div class="card card-body{} pb-1">{}</div></div>\
 """.format(the_id, template.subject_as_html(trim=True), the_id, classname, template.content_as_html())
-
```

Comment:
We didn't talk about it, but are we thinking this function is just too big to go in misc.py?